web/stats.php

```diff
@@ -32,6 +32,22 @@
     $previous_year = 'FYMP';
 
 $years = array('2005', '2006', '2007', '2008', '2013', '2014');
+
+# this is so we get the mapit IDs and cons names for the year and
+# hence can get the MP for the year from DaDem and not the current
+# on
+$mapit_year_args = array(
```

Comment:
Better to do this with range checks? Then you won't have to add anything for 2015.